tests/tests.sql

```diff
@@ -18,130 +36,35 @@ insert into db.people_cris (crash_id, unit_nbr, prsn_nbr, is_primary)
 values (1, 2, 1, true);
 insert into db.people_cris (crash_id, unit_nbr, prsn_nbr, is_primary)
 values (1, 2, 2, false);
-
--- verify that cris records match inserts
-select * from people_cris_view;
--- verify that vz records are blank except for IDs
-select * from people_vz_view;
--- verify that unified records match cris
-select * from people_unified_view;
-
 -------------------------------
--- 2. Update crashes
+-- 3. Update crashes
 -------------------------------
--- set crash road type to 3 - arterial
-update db.crashes_cris set road_type_id = 3 where crash_id = 1;
-
--- verify that cris crash id 1 has road_type_id 3 (arterial)
-select * from people_cris_view;
--- verify that vz records are blank except for IDs
-select * from people_vz_view;
--- verify that unified records have road type 3 (arterial)
-select * from people_unified_view;
-
--- override cris road type by setting vz to 4 (highway)
-update db.crashes_vz set road_type_id = 4 where crash_id = 1;
-
--- verify that cris crash id 1 has road_type_id 3 (arterial)
-select * from people_cris_view;
--- verify that vz records have road_type_id 4 (highway)
-select * from people_vz_view;
--- verify that unified records have road type 4 (highway)
-select * from people_unified_view;
-
--- revert vz road type edit
-update db.crashes_vz set road_type_id = null where crash_id = 1;
-
--- verify that cris crash id 1 has road_type_id 3 (arterial)
-select * from people_cris_view;
--- verify that vz records have are blank except for IDS
-select * from people_vz_view;
--- verify that unified records have road type 4 (arterial)
-select * from people_unified_view;
-
-
+-- moves location to location_id 
+update db.crashes_cris set latitude = 30.16290408, longitude = -97.65251883 where crash_id = 1;
+-- verify that unified record has location ID 6BDBB50336
+select * from people_unified_view where crash_id = 1;
+-- override cris lat lon
+update db.crashes_cris set latitude = 30.35808625, longitude = -97.68813396 where crash_id = 1;
+-- verify that unified record has location ID 153D618A97
+select * from people_unified_view where crash_id = 1;
+-- verify 1 insert event and two update events in the change log
+select * from db.change_log where record_type = 'crashes' and record_id = 1;
 -------------------------------
--- 2. Update units
+-- 4. Update units
 -------------------------------
 -- update one cris unit to be bicycle 
--- (we are simulating an import here by using crash_id + unit_nbr)
 update db.units_cris set unit_type_id = 5 where crash_id = 1 and unit_nbr = 1;
```

Comment:
i see that u are updating by the compound of crash id and unit nbr instead of the custom unit id... dont want to presume here but if this is what felt more natural then maybe that argues in favor of continuing w our use of these compound primary keys instead of adding custom ids to each table heh :stuck_out_tongue_winking_eye: